src/pages/cocktail/[id]/index.js

```diff
@@ -0,0 +1,142 @@
+import {
+  Container,
+  Typography,
+  Button,
+  CardMedia,
+  Card,
+  CardContent,
+  Box,
+  CircularProgress,
+} from '@material-ui/core';
+import { useRouter } from 'next/router';
+import { useEffect, useState } from 'react';
+import Header from '../../../components/Header';
+
+function Redirect({ to }) {
+  const router = useRouter();
+
+  useEffect(() => {
+    router.push(to);
+  }, [router, to]);
+
+  return null;
+}
+
+export default function CocktailPage({ drinkData }) {
+  const [shouldRedirect, setShouldRedirect] = useState(false);
+  const { isFallback } = useRouter();
+
+  if (shouldRedirect) {
+    return <Redirect to="/" />;
+  }
+
+  return (
+    <Container maxWidth="md">
+      {isFallback ? (
+        <>
+          <Typography variant="h6" color="primary">
+            Loading...
+          </Typography>
+          <br />
+          <CircularProgress />
+        </>
+      ) : (
+        <>
+          <Header subtitle={`Cocktail ${drinkData.strDrink}`} />
+          <Card
+            style={{
+              display: 'flex',
+              alignItems: 'center',
+              backgroundColor: '#FFEFD5',
+              marginTop: '30px',
+            }}
+          >
+            <CardMedia
+              component="img"
+              sx={{ height: 140 }}
+              title={`cocktail ${drinkData.strDrink}`}
+              image={`${drinkData.strDrinkThumb}/preview`}
+            />
+            <CardContent>
+              <Typography
+                style={{ display: 'inline-block' }}
+                variant="h6"
+                color="primary"
+              >
+                Best served in:
+              </Typography>{' '}
```

Comment:
are `{' '}` necessary?

Reply:
empty space as in-line block to separate both strings